recipes/state-machine-cat/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "state-machine-cat" %}
+{% set version = "v10.1.10" %}
```

```suggestion
{% set version = "v11.0.0" %}
```
v11.0.0 is the latest version (I noticed I haven't explicitly  documentet that  on  on GitHub yet - but it is the latest  there as well.

This is assuming the `node` version conda forge uses is >14

---

Yes, based on the ci logs, I updated the version and enforced `node >14` - everything looks good (based on the logs it uses `node 18.*`